features/step_definitions/basic_steps.rb

```diff
@@ -420,12 +420,23 @@ def assert_link_exists path, text
   find("#attendance_checkbox", visible: false).trigger('click')
 end
 
+
+When(/^I scroll to the bottom of the page$/) do
+  page.execute_script "window.scrollBy(0,10000)"
+end
+
 Then(/^I should( not)? see "([^"]*)" within "([^"]*)"$/) do |negated, project_title, project_list_area|
   within("##{project_list_area}") do
     if negated
       expect(page).to_not have_content(project_title)
```

Comment:
Never assert that sth. is not visible. This will always be true, e.g. if the page is not loaded. If you really have to, assert that sth. must be visible before checking that sth. is not visible. Right now, I don't know if `within('#id')` fulfills that.